docs/SRS-Volere/SRS.tex

```diff
@@ -656,7 +656,13 @@ \subsection{Understandability and Politeness Requirements}
     determine next steps.
 \end{enumerate}
 \subsection{Accessibility Requirements}
-TBD may be in constraints.
+\begin{enumerate}[{UH-AS}1.]
+  \item The application must adhere to accessibility policies set by the City Of
+    Hamilton \href{https://www.hamilton.ca/people-programs/inclusion-diversity-equity-accessibility/accessibility-services/accessibility#policies-procedures}{Linked Here}.\\
```

Comment:
This line is over 80, but the hyperlink won't work if its split on multiple lines

Reply:
is the linter not allowing the commit because of that? It let my hyperlinks through when Kyle set it up

Reply:
yes it does allow the hyperlinks it just gives a warning for that

Reply:
I'll fix the formatting

Reply:
Where is the ruleset? Everyone should have the same ruleset on the formatter to prevent these issues

Reply:
it's just the default for tex-fmt. If you wanna install do `cargo install tex-fmt`